### DISCUSSION TITLE
noobaa version upgrade process

### DISCUSSION THREAD

**Discussion by Alansyf** [noobaa/noobaa-operator#1164] [Q&A]
Hi, 

I understand by using `noobaa install` to install noobaa from scratch; 
But we didn't see options for noobaa upgrade. 

How or what procedure we should follow to take noobaa upgrade?

**Comment by dannyzaken** [comment]
Hi @Alansyf, we currently don't have a way to upgrade noobaa using the CLI automatically. Hopefully, we will add this option soon.

You can upgrade it manually, but I can't guarantee a smooth upgrade. To do that, you will need to use the CLI of the new version and generate a yaml  with updated resources definition:
```
noobaa install yaml [flags for install if any] > install.yaml
```
Review the yaml and apply it:
```
kubectl apply -f install.yaml
```

As mentioned before, this process is not tested and hopefully, we will have an automated command for it in the CLI soon. I suggest testing it before applying it to any production system.

**Reply by Alansyf** [reply]
Thanks @dannyzaken , do you have any ticket for working on this "upgrade" CLI so I can watch that.
